.jenkins

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env groovy
 
 // Include this shared CI repository to load script helpers and libraries.
-library identifier: 'vapor@1.0.0-RC13', retriever: modernSCM([
+library identifier: 'vapor@1.20.1', retriever: modernSCM([
```

Comment:
Sorry - I think I broke this one with the emulator changes I made last week. I can cut a branch to try to fix it.

Change library identifier to vapor@1.20.2. add 'skipUnitTest': true, as here and it should work: https://github.com/vapor-ware/synse-snmp-plugin/blob/master/.jenkins